------------------------------------------------------------
revno: 250
fixes bug(s): https://launchpad.net/bugs/617400
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Fri 2010-08-13 10:07:14 -0400
message:
  fix 'seedfrom' in DataSourceNoCloud (LP: #617400)
=== modified file 'cloudinit/DataSourceNoCloud.py'
--- old/cloudinit/DataSourceNoCloud.py	2010-08-12 16:49:29 +0000
+++ new/cloudinit/DataSourceNoCloud.py	2010-08-13 14:07:14 +0000
@@ -77,12 +77,12 @@
         # attempt to seed the userdata / metadata from its value
         if "seedfrom" in md:
             seedfrom = md["seedfrom"]
-            found = False
+            seedfound = False
             for proto in self.supported_seed_starts:
                 if seedfrom.startswith(proto):
-                    found=proto
+                    seedfound=proto
                     break
-            if not found:
+            if not seedfound:
                 cloudinit.log.debug("seed from %s not supported by %s" %
                     (seedfrom, self.__class__))
                 return False

